docker/pravega/entrypoint.sh

```diff
@@ -48,9 +50,23 @@ configure_tier2() {
         done
     fi
     add_system_property "filesystem.root" "${NFS_MOUNT}"
+    ;;
+
+    HDFS)
     add_system_property "hdfs.hdfsUrl" "${HDFS_URL}"
     add_system_property "hdfs.hdfsRoot" "${HDFS_ROOT}"
     add_system_property "hdfs.replication" "${HDFS_REPLICATION}"
+    ;;
+    EXTENDEDS3)
+    EXTENDEDS3_ROOT=${EXTENDEDS3_ROOT:-"/"}
+    add_system_property "extendeds3.root" "${EXTENDEDS3_ROOT}"
```

Comment:
The line above this checks and sets this var to default value if it is not defined.